packages/styles/src/calendar.scss

```diff
@@ -411,8 +422,60 @@ $fd-calendar-special-days: (
     @include fd-reset();
 
     display: flex;
+
+    &--legend {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(7.5rem, 1fr));
+      grid-gap: 0.5rem;
+      margin-top: 0.5rem;
```

Comment:
if this is not part of the table, like I suggested in another comment, you can achieve this with flex. The reason to use flex here is because with modifier classes you can easily achieve different variations of the legend. It can be the way you did it, but it can also be like this:
![Screenshot 2024-10-21 at 2 50 18 PM](https://github.com/user-attachments/assets/0d6ebe10-ff34-474a-93ca-5dfbbeed4554)